submissions/blessingOlusuji/HTML/product.html

```diff
@@ -0,0 +1,72 @@
+<!Doctype html>
+<html lang ="en">
+<head>
+    <meta charset ="UTF-8">
+    <title>Spotless-B Mobile Page</title>
+</head>
+<body>
+   <h2><i>Spotless-B Mobile</i></h2>
+   <nav>
+      <ul>
+        <li><a href ="#">Home</a></li>
+        <li><a href ="#">Cart</a></li>
+        <li><a href ="#">About us</a></li>
+        <li><a href ="#">Contact us</a></li>
+      </ul>
+   </nav>
+
+<!--This session is about company's description-->
+<session>
+       <article>
+           <p>
+              Spotless-B Mobiles is one of the best phone namufacturing company in the world. 
+              Our goal is to provide affordable and easy to use mobile gadgets for everyone,
+              our products are built to solve real-life problems and are built to meet individual preferences.
+          </p>
+      </article>
+ </session>
+ <hr>
+ <!---This session is about product name and image-->
+ <session>
+      <div>
+          <img src="images/mobile1.jpg" width="300px" height ="300px">
+          <h4><u>Nokia Edge</u> : <strong>#110,000</strong></h4>
+          <p> Nokia Edge smartphone runs on Android v7.0 (Nougat) operating system. <br>
+              The phone is powered by octa core (2 GHz, quad core, cortex A53 + 1.2 GHz,<br>
+              quad core, cortex A53) processor. It runs on the mediaTek MT6755 chipset. <br>
+              It has 3GB RAM and 32 GB internal storage.
+          </P>
+       </div>
+       <div>
+           <img src="images/set1.jpg" width="300px" height ="300px">
+           <h4><u>Iphone7</u> : <strong>#150,000</strong></h4>
+           <p> Phone7 is the good phone with fast processor, water resistance and durable skin.<br>
+               In this, have good cameras with rear optical image stabilisation.
+           </P>
+      </div>
+</session>
+<!---This session is for similar products-->
+<session>
+       <div>
+           <h3>Check out other similar Products:</h3>
+           <ol>
+              <li> Iphone 8</li>
+              <li> Iphone 10</li>
+              <li> Noke Edge Max</li>
+           </ol>
+      </div>
+</session>
+<!---footer session-->
+<footer>
+   <div>
+        <ul>
+            <li><a href ="#"> Reviews</a></li> 
+            <li><a href ="#"> FAQs</a></li> 
+            <li><a href ="#"> Blogs</a></li> 
+       </ul>
+       <p>&copy:Spotless-B 2020;</p>
+  </div>
+</footer>
+   
```

Comment:
Remove whitespace

Reply:
Please, what do you mean by white space here?

Reply:
The empty line..

Reply:
Okay, I just checked but there isnt any, however, I have closed up the lines between closing body and html tag. I will be pushing modified version now, thanks.